.gitignore

```diff
@@ -34,6 +34,9 @@ mergelog.txt
 *.user
 *.sln.docstates
 
+# NT.cmd
+nt.cmd
+
```

Comment:
let's not commit this. as we discussed this morning with scott, let's figure out how to incorporate what this script does into our workflow... 
for now, i keep this script up one level (in the parent directory) to avoid this.